app/components/project-sorts.cjsx

```diff
@@ -0,0 +1,9 @@
+module.exports =
```

Comment:
I have provided an update for this file, this PR will update when https://github.com/zooniverse/Panoptes-Front-End/pull/2370 is merged